### DISCUSSION TITLE
Conda Installation - Horovod Error

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2107] [Q&A] [answered]
Hi,

Installed DeePMD-kit via conda. Getting errors when trying to do parallel training.
![conda_error](https://user-images.githubusercontent.com/10074710/202688522-bed70401-9e91-4dd1-a044-31fedac5bbc4.png)

I checked the conda list also horovod and mpich are there.

**Comment by njzjz** [comment, marked as answer]
You can check if packages with the "external" tag are installed. Those packages don't contain an actual MPI execute.

https://anaconda.org/anaconda/mpich/files

![image](https://user-images.githubusercontent.com/9496702/202879204-37f82597-65a2-4d0f-a62b-f145715d9a66.png)

**Reply by AnuragKr** [reply]
No, the Above was not installed. The following packages were installed --

![Screenshot 2022-11-21 at 11 18 14 AM](https://user-images.githubusercontent.com/10074710/202978776-4c7be073-0f35-4dd0-9bf9-f587a1af21d7.png)

The issue came after I ran the conda update command.  For the time being, I solved the issue by doing a re-installation.
